src/utils/isStandardSyntaxMediaFeature.js

```diff
@@ -16,5 +16,8 @@ export default function (mediaFeature) {
   // SCSS or Less interpolation
   if (hasInterpolation(mediaFeature)) { return false }
 
+  // SCSS variables
+  if (/\$.+?/.test(mediaFeature)) { return false }
```

Comment:
@evilebottnawi Thanks for the info. 

> So: What if we just outlaw strings that contain variables *and* an operator?

Shall we go this instead then i.e. making it a little less restrictive?